### PR TITLE
Add server side new user storage

### PR DESCRIPTION
**Pull request by sammyson101**
Currently hard codes user ID and Difficulty level, these will get changed in the future

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -23,32 +21,16 @@ addMiddlewares(app);
 // configure routes
 app.use('/trails', trailsRouter);
 
-app.use(express.static('db'));
+//app.use(express.static('db'));
```
Comment (mgarber93) [inline src/index.js]: Can we remove comments

File: src/routes/newUser.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  const newUser = new User(
+    2,
+    req.body.username,
+    req.body.password,
+    req.body.email,
+    req.body.displayName,
+    3,
+  );
+
+  users.pushUser(newUser);
```
Comment (mgarber93) [inline src/routes/newUser.js:21]: To be clear this wont be persisted between server sessions?

**Review by mgarber93 — APPROVED** [review]
I'd remove comments but lgtm

File: src/routes/newUser.js (hunk 2 above)
Comment (sammyson101) [reply src/routes/newUser.js:21]: Correct. When I restarted the server it lost all accounts I had pushed. If we want to persist beyond that we have to have a file off the server somewhere to hold are accounts instead. Refactoring for that should be pretty easy though.

**Pull request merged**